netlify.toml

```diff
@@ -13,4 +13,9 @@ force = true
 [[redirects]]
 from = "https://blog.opentelemetry.io/*"
 to = "https://opentelemetry.io/blog/:splat"
-force = true
\ No newline at end of file
+force = true
+
+[[headers]]
+  for = "/schemas/:version"
+  [headers.values]
+    content-type = "application/x-yaml"
```

Comment:
I think `application/yaml` is the the latest accepted media type.

From https://www.ietf.org/archive/id/draft-ietf-httpapi-yaml-mediatypes-00.html#name-media-type-application-yaml

>Deprecated alias names for this type: application/x-yaml

Reply:
I've found a more recent (and non-expired) draft: https://www.ietf.org/archive/id/draft-ietf-httpapi-yaml-mediatypes-10.html, dated 2023/08/30. This is still a draft, but I'll switch to `application/yaml` as you suggest. Thanks for catching that!